staging/src/k8s.io/apiserver/pkg/authorization/config/helpers.go

```diff
@@ -15,17 +15,3 @@ limitations under the License.
 */
```

Comment:
squash the `Update API types and fix lint errors` commit into the `Bootstrap API Types` commit (since it's just fixup)